src/modules/CardCart.js

```diff
@@ -0,0 +1,7 @@
+function displayNumberOfCard() {
+  const getNumberOfCard = document.querySelector('.CardNumber');
+  const card = document.querySelectorAll('.card');
+  getNumberOfCard.innerHTML = `( ${card.length} )`;
+}
+
```

Comment:
Great job team🤝

- [ ] As stated in the project requirements, we are supposed to use ES6 syntax instead of using the function keyword. Kindly use ES6 syntax for all your functions. You may implement this across all your files like `involvementapi.js`, and `LikeEvent.js`.